gdcdatamodel/validators/graph_validators.py

```diff
@@ -12,8 +12,7 @@ class GDCGraphValidator(object):
     def __init__(self):
         self.schemas = gdcdictionary
         self.required_validators = {
-            'links_validator': GDCLinksValidator(),
-            'uniqueKeys_validator': GDCUniqueKeysValidator()
+            'links_validator': GDCLinksValidator()
```

Comment:
(trivial tip) keeping the comma of the last item would efficiently [reduce VCS diff size](https://github.com/ambv/black#trailing-commas)